lenses_python/ConnectorHandler.py

```diff
@@ -11,7 +12,7 @@ def __init__(self, url, username, password, token):
         self.token = token
         self.url_extend = CONNECTOR_END_POINT  # /api/proxy-connect/
         self.default_headers = {'Content-Type': 'application/json', 'Accept': 'application/json',
-                            'x-kafka-lenses-token': self.token}
+                                'x-kafka-lenses-token': self.token}
```

Comment:
better to have this kind of format is more readable and you will avoid also the extra space you have added:

```python
{
   'Content-Type': 'application/json', 'Accept': 'application/json',
   'x-kafka-lenses-token': self.token
}
```